### DISCUSSION TITLE
Add demo API Token

### DISCUSSION THREAD

**Discussion by GunniBusch** [Performave/convoy#81] [Feature Requests & Suggestions]
add demo/test api for developing a service that uses convoy without creating live things.

**Comment by ericwang401** [comment]
If you could provide more details about how to provide this in a secure/cost effective solution, please do. Because it seems like a huge security risk to let random users provision virtual machines that can run anything or just interact with the Proxmox node in general.

**Reply by GunniBusch** [reply]
I think there was a misunderstanding. 
if I deploy my own convoy panel, I have Access to a api. But when I want to develop a app, that access this api, I can only use a live api key. Therefore a idea would be to add a demo mode to your own instance so you can develop safely without creating eg. servers. Basically like the test api of stripe.

**Reply by ericwang401** [reply]
I don't think this would be a feasible idea. It would increase the complexity in the code across the board and require plenty of refactoring. It's also difficult because Convoy relies on Proxmox, so proper API responses from Proxmox is needed for Convoy to function.
